pkg/charts/template-values.go

```diff
@@ -87,5 +90,36 @@ type (
 	Identity struct {
 		TrustAnchorsPEM string
 		TrustDomain     string
+		Issuer          *Issuer
+	}
+
+	// Issuer has the Helm variables of the identity issuer
+	Issuer struct {
+		ClockSkewAllowance  string
+		IssuanceLifetime    string
+		CrtExpiryAnnotation string
+		CrtExpiry           time.Time
+		TLS                 *TLS
+	}
+
+	// ProxyInjector has all the proxy injector's Helm variables
+	ProxyInjector struct {
+		*TLS
+	}
+
+	// ProfileValidator has all the profile validator's Helm variables
```

Comment:
similar question to earlier, what's the intent of wrapping `TLS` in a new struct like `ProfileValidator`?